package.json

```diff
@@ -77,7 +77,8 @@
       "src/**/*.{js,jsx}"
     ],
     "coveragePathIgnorePatterns": [
-      "/node_modules/"
+      "/node_modules/",
+      "<rootDir>/src/.*/container.jsx"
```

Comment:
This is to ignore the container components, which have a mapStateToProps, a mapDispatchToProps, and a call to Redux's connect HOC.